config/sync/.htaccess

```diff
@@ -0,0 +1,23 @@
+# Deny all requests from Apache 2.4+.
```

Comment:
Are the changes in ```/config``` really required? It feels like something we can skip in this PR.